### ISSUE TITLE
Feature: fonts sizes need to be bigger (selectable preference)

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#45]
This is an important one.  Lots of people can't read the default text size.
One idea: add View > Font Size > xx-small|x-small|small|medium|large|x-large|xx-large  (just like CSS).

Bob Elling also suggests using Cmd-Plus/Equal and Cmd-Minus (which is the browser standard for changing font size on the fly).  This is a good idea, but right now that combination is used for Speed Up/Slow Down.

Making it a View menu item allows us to also change margins and padding accordingly.

This setting should be persistent on a per-user basis (normal Preference).

**Comment by mpogue2** [comment]
Tagging this one SUPER ANNOYING, because it's very hard for some to read the default text size.

**Comment by mpogue2** [comment]
First part of this is in as bdfb12daaf43bc8495134a466fe42375118b200a .

Cmd-8 is bound to View > Zoom > Zoom In, Cmd-9 is View > Zoom > Zoom Out, Cmd-0 is View > Zoom > Reset (which resets back to the default sizes).  

I've introduced some minor UX bugs in the process, which I'll file shortly.  But, as-is this should be 95% usable for those who prefer larger fonts.

Note: getting the search fields to scale properly again was gnarly.  Qt has a bug where it doesn't track the width of the first column properly, and I needed to work around that.
